### PR TITLE
Start abstract submission phase

### PR DESCRIPTION
**Pull request by pmandler**
This needs the backend to support starting the number at 161.

### CODE REVIEW

File: venues/MIDL.io/2019/Conference/webfield/programchairAbstractWebfield.js [hunk 1]
```diff
@@ -0,0 +1,796 @@
+
+// Assumes the following pattern for meta reviews and official reviews:
+// CONFERENCE + '/-/Paper' + number + '/Meta_Review'
+// CONFERENCE + '/-/Paper' + number + '/Official_Review'
+
+// Constants
+var HEADER_TEXT = 'Program Chairs Console';
```
Comment (melisabok) [inline venues/MIDL.io/2019/Conference/webfield/programchairAbstractWebfield.js]: Is there a reason you need to user another pc console than the one in the builder?

File: venues/MIDL.io/2019/Conference/webfield/areachairWebfieldAbstract.js [hunk 2]
```diff
@@ -0,0 +1,699 @@
+
+// Assumes the following pattern for meta reviews and official reviews:
+// CONFERENCE + '/-/Paper' + number + '/Meta_Review'
```
Comment (melisabok) [inline venues/MIDL.io/2019/Conference/webfield/areachairWebfieldAbstract.js]: Is there a reason to overwrite the ac console here?

Comment (pmandler) [reply venues/MIDL.io/2019/Conference/webfield/areachairWebfieldAbstract.js]: The ACs are going to act as reviewers for the abstract submissions.

Comment (melisabok) [reply venues/MIDL.io/2019/Conference/webfield/areachairWebfieldAbstract.js]: so, you are setting the reviewer console in the ac console?

File: venues/MIDL.io/2019/Conference/webfield/programchairAbstractWebfield.js (hunk 1 above)
Comment (pmandler) [reply venues/MIDL.io/2019/Conference/webfield/programchairAbstractWebfield.js]: I'll remove this file! Yay.

**Pull request merged**